proto/panacea/oracle/v2alpha2/tx.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+package panacea.oracle.v2alpha2;
+
+option go_package = "github.com/medibloc/panacea-core/x/oracle/types";
+
+import "panacea/oracle/v2alpha2/oracle.proto";
+import "panacea/oracle/v2alpha2/genesis.proto";
+import "gogoproto/gogo.proto";
+
+// Msg defines the Msg service.
+service Msg {
+  // RegisterOracle defines a method for registration of oracle.
+  rpc RegisterOracle(MsgRegisterOracle) returns (MsgRegisterOracleResponse);
+
+  // VoteOracleRegistration  defines a method for voting registration of oracle.
+  rpc VoteOracleRegistration(MsgVoteOracleRegistration) returns (MsgVoteOracleRegistrationResponse);
+}
+
+// MsgRegisterOracle defines the Msg/RegisterOracle request type.
+message MsgRegisterOracle {
+  OracleRegistrationInfo oracle_registration_info = 1;
+}
```

Comment:
How about deleting OracleRegistrationInfo and using it like below?

```suggestion
message MsgRegisterOracle {
  string unique_id = 1;
  string oracle_address = 2;
  bytes node_pub_key = 3;
  bytes node_pub_key_remote_report = 4;
  int64 trusted_block_height = 5;
  bytes trusted_block_hash = 6;
}
```

Reply:
We can delete and modify as you suggested.

However, the reason why I make a message `OracleRegistrationInfo` is it would be useful when we handle the `MsgRegisterOracle` in msg server(in `msg_server_oracle.go`).

Reply:
I got the point that @gyuguen said. I fixed it in [08c9c9f](https://github.com/medibloc/panacea-core/pull/361/commits/08c9c9f9b88dc30e411e48adfec488a1753f489c)